scalability_test/create_notebook.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ $# -ne 3 ]; then
+    echo "Usage: $0 <num_notebooks> <batch_size> <username>"
+    exit 1
+fi
+
+# Number of notebbooks to create
+num_notebooks=$1
+# Number of requests to make concurrently
+batch_size=$2
+username=$3
+SECONDS=0
+
+create_resources() {
+    notebook_name=$1
+    # Create service account
+    sed "s/ope-placeholder/$notebook_name/g" sa_template.yaml | oc apply -f -
```

Comment:
It doesn't appear to be necessary to create the serviceaccount -- this is created automatically in response to creating the notebook (and will be deleted automatically when the notebook is deleted).

Reply:
You are right. I double checked that service account is created/deleted automatically.